decidim-accountability/app/permissions/decidim/accountability/admin/permissions.rb

```diff
@@ -28,6 +29,10 @@ def timeline_entry
           @timeline_entry ||= context.fetch(:timeline_entry, nil)
         end
 
+        def projects_import
+          @projects_import ||= context.fetch(:projects_import, nil)
```

Comment:
I believe this "resource" is never defined based on the permission checks so we shouldn't need this method?